BloodManagmentSystem/Persistance/EntityConfigurations/BloodRequestConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using BloodManagmentSystem.Core.Models;
+using System.Data.Entity.ModelConfiguration;
+
+namespace BloodManagmentSystem.Persistance.EntityConfigurations
+{
+    public class BloodRequestConfiguration : EntityTypeConfiguration<BloodRequest>
+    {
+        public BloodRequestConfiguration()
+        {
+            Property(br => br.BloodType)
+                .IsRequired();
+
+            Property(br => br.DueDate)
+                .IsRequired();
+
+            Property(br => br.City)
+                .HasMaxLength(50);
```

Comment:
It might be useful to check `MinLength`

Reply:
This class is responsible for configuring table fields, so you can't set min length ;)